subprojects/configuration-cache/src/test/kotlin/org/gradle/configurationcache/DefaultIgnoredConfigurationInputsTest.kt

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.configurationcache
+
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Test
+import java.io.File
+
+
+class DefaultIgnoredConfigurationInputsTest {
```

Comment:
Maybe add some tests, where ignores are specified with backslashes (`\`)?

Reply:
Do you think we should interpret backslashes in the pattern as separators only on Windows machines, or treat them as such on UNIX-like ones, too?